errbot/backends/telegram_messenger.py

```diff
@@ -339,6 +340,121 @@ def prefix_groupchat_reply(self, message, identifier):
         super().prefix_groupchat_reply(message, identifier)
         message.body = '@{0}: {1}'.format(identifier.nick, message.body)
 
+    def _telegram_special_message(self, chat_id, content, msg_type, **kwargs):
+        """Send special message."""
+        if msg_type == 'document':
+            msg = self.telegram.sendDocument(chat_id=chat_id,
+                                             document=content,
+                                             **kwargs)
+        elif msg_type == 'photo':
+            msg = self.telegram.sendPhoto(chat_id=chat_id,
+                                          photo=content,
+                                          **kwargs)
+
+        elif msg_type == 'audio':
+            msg = self.telegram.sendAudio(chat_id=chat_id,
+                                          audio=content,
+                                          **kwargs)
+
+        elif msg_type == 'video':
+            msg = self.telegram.sendVideo(chat_id=chat_id,
+                                          video=content,
+                                          **kwargs)
+        elif msg_type == 'sticker':
+            msg = self.telegram.sendSticker(chat_id=chat_id,
+                                            sticker=content,
+                                            **kwargs)
+        elif msg_type == 'location':
+            msg = self.telegram.sendLocation(chat_id=chat_id,
+                                             latitude=kwargs.pop('latitude', ''),
+                                             longitude=kwargs.pop('longitude', ''),
+                                             **kwargs)
```

Comment:
I think we should add a final `else:` clause here which raises an exception, saying an invalid `msg_type` was specified. Otherwise the line below is going to raise a very confusing error because `msg` is not defined.